bundles/test/org.eclipse.smarthome.magic/ESH-INF/thing/thing-types.xml

```diff
@@ -101,4 +102,29 @@
 			<channel id="number" typeId="number" />
 		</channels>
 	</thing-type>
+
+	<thing-type id="firmware-update">
+		<label>Magic Firmware Updatable Thing</label>
+		<description>A thing that can be firmware updated.</description>
```

Comment:
How about `A thing which allows updating its firmware`?

No trailing `.` anyway, please.

Reply:
Your description looks more accurate, I will use it.